tools/llm_bench/benchmark.py

```diff
@@ -129,6 +129,13 @@ def get_argprser():
     parser.add_argument('-od', '--output_dir', help='Save the input text and generated text, images to files')
     llm_bench_utils.model_utils.add_stateful_model_arguments(parser)
     parser.add_argument("--genai", action="store_true", help="Use OpenVINO GenAI optimized pipelines for benchmarking")
+    parser.add_argument(
+        "--lora",
+        nargs='*',
+        required=False,
+        default=None,
+        help="Path to LoRA adapters for using OpenVINO GenAI Text2Image optimized pipelines with LoRA for benchmarking")
```

Comment:
LoRA can be applied for LLM pipelines as well (and for VLM and Whisper in next releases)
Should it be a generic parameter? not only for Text2Image